Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -222,8 +222,13 @@ private void DetermineOneRowMode()
 
         private void RecalculateLayout(double containerWidth)
         {
+            var itemsWrapGridPanel = ItemsPanelRoot as ItemsWrapGrid;
```

Comment:
My understanding is that the AdaptiveGridVIew requires ItemsWrapGrid to work per [this](https://github.com/Microsoft/UWPCommunityToolkit/blob/c91fa094740830c33e8f4570e3ff806e2e18f387/Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs#L170). Ideally I'd like to get rid of this control and replace it with a custom panel, but who has the time :)